### PR TITLE
Feature/14 read only api routes

### PR DESCRIPTION
**Pull request by Alexander-Prime**
I decided to try a new directory structure on the API that I wrote up in the summary of #14. The goal is to separate code that handles server requests (routes & controllers) from code that reads and writes to the database (models & services).

### CODE REVIEW

File: api/src/server/documents/sheets/cells/controller.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Request } from "hapi";
+
+import * as Cell from "../../../../data/Cell/service";
```
Comment (MarkHalls) [inline api/src/server/documents/sheets/cells/controller.ts:3]: Do we not have a better way to get imports from nested directories? This feels fragile.

Comment (Alexander-Prime) [reply api/src/server/documents/sheets/cells/controller.ts:3]: I agree, though my first attempt generated missing module errors. I'll give it another shot.

Comment (MarkHalls) [reply api/src/server/documents/sheets/cells/controller.ts:3]: Should be OK for now and can be addressed later for the project as a whole.

**Review by MarkHalls — APPROVED** [review]
Looks good. I'll be adding an issue to update the import directory nesting.

**Pull request merged**